DAV 1.3.3 build failure

```
    [4 of 4] Compiling Main

    /var/stackage/work/unpack-dir/unpacked/DAV-1.3.3-3256cc44d90d12940519bc0674ea9a92ff7d9858935fda1bb0e85
1fcc39c82de/hdav.hs:35:1: error:
        Could not find module ‘Network’
        Use -v (or `:set -v` in ghci) to see a list of the files searched for.
       |
    35 | import Network (withSocketsDo)
       | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

cc @clinty

--- a/build-constraints.yaml
+++ b/build-constraints.yaml
@@ -1163,7 +1163,7 @@ packages:
         - hOpenPGP < 0 # via bzlib
         - openpgp-asciiarmor
         - MusicBrainz
-        - DAV < 0 # https://github.com/commercialhaskell/stackage/issues/5054
+        - DAV
         - hopenpgp-tools < 0 # via graphviz
         - opensource
         - debian < 0 # via bzlib

